src/components/Tasks/index.jsx

```diff
@@ -42,6 +52,11 @@ function Tasks({ isTaskCreating, onTaskCreation }) {
         );
     });
 
+    useEffect(() => {
+        reset();
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
can you explain we are using this comment here?

Reply:
I've used the comment here to disable eslint for the next line because of getting a warning. I've given the screenshot below. Please suggest me what is the good practice here. Thank you.

![Screenshot 2023-09-24 at 9 59 23 AM](https://github.com/sse-wtag/todo-app-react/assets/144016889/3534acdc-ac27-4b5c-9bb3-8fd8df1fdb9b)